packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/CompleteProfile/selectors.ts

```diff
@@ -14,41 +13,36 @@ type ReturnData = {
 }
 
 export const getData = (state: RootState): ReturnData => {
-  let currentStep = 0
-  let isVerifiedId = false
-  let isKycPending = false
-  let isBankOrCardLinked = false
-  let isBuyCrypto = false
   const isKycStateNone =
-    // @ts-ignore
-    selectors.modules.profile.getUserKYCState(state).getOrElse('') === 'NONE'
+    selectors.modules.profile.getUserKYCState(state).getOrElse('') === KYC_STATES.NONE
 
   const isFirstLogin = selectors.signup.getFirstLogin(state)
 
   if (isFirstLogin || isKycStateNone) {
     return {
-      currentStep,
+      currentStep: 0,
       isBankOrCardLinked: false,
       isBuyCrypto: false,
       isKycPending: false,
       isVerifiedId: false
     }
   }
 
-  const userDataR = selectors.modules.profile.getUserData(state)
-  const userData = userDataR.getOrElse({
+  const userData = selectors.modules.profile.getUserData(state).getOrElse({
+    kycState: undefined,
     tiers: { current: 0 }
-  } as UserDataType)
+  })
 
-  const { KYC_STATES } = model.profile
-  isKycPending =
+  let currentStep = 0
```

Comment:
how come this was moved to here instead of above line 21?

Reply:
Just for historical purposes - at line 22 we are always returning 0 if the conditions are met, and then below we do the +1 according to the user state. But all of this is going to be deprecated soon by a few new endpoints introduced by the Backend team